tests/wpt/metadata/XMLHttpRequest/send-content-type-charset.htm.ini

```diff
@@ -18,3 +18,9 @@
   [If multiple charset parameters are given, all should be rewritten]
     expected: FAIL
 
+  [Correct text/plain MIME with charset]
```

Comment:
We reformat content-type strings, which isn't forbidden by the spec. We don't want to regex-manipulate content-types